### PR TITLE
First assignment

### PR DESCRIPTION
**Pull request by aliskin**

### CODE REVIEW

File: aliskin/assignment1/anagram.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from collections import defaultdict
+
+def isAnagram(s, t, case_sensitive=True):
```
Comment (ariava) [inline aliskin/assignment1/anagram.py]: Please use the official function naming style: lowercase words separated by underscores.
See https://www.python.org/dev/peps/pep-0008/#function-and-variable-names.

File: aliskin/assignment1/anagram.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    """
+    Given two strings s and t returns True if t is anagram of s and False otherwise.
+    """
+    if len(s) != len(t):
+        return False
+    if (not case_sensitive):
+        s = s.casefold()
+        t = t.casefold()
+    symbols = defaultdict(int)
+    for c in s:
```
Comment (ariava) [inline aliskin/assignment1/anagram.py]: You can also use a collections.Counter object to count occurrences of characters in words: https://docs.python.org/2/library/collections.html#collections.Counter

**Review by ariava — CHANGES_REQUESTED** [review]
Nice job! Please have a look at my comments.

File: aliskin/assignment1/kth-element.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+class ListNode:
+    def __init__(self, x):
+        self.value = x
+        self.next = None
+
+def kth_element(head, k):
```
Comment (ariava) [inline aliskin/assignment1/kth-element.py]: It's "kth_last_element", that is the kth element starting from the last one going backwards.

**Comment by aliskin** [comment]
Arianna, thanks for your comments!
I'm not sure if I updated the pull request properly.. hope it works!

File: aliskin/assignment1/anagram.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+from collections import defaultdict
+from collections import Counter
+
+def is_anagram(s, t, case_sensitive=True):
```
Comment (ariava) [inline aliskin/assignment1/anagram.py]: Can you add a few examples to show that the function really works? :)

File: aliskin/assignment1/kth-element.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
+class ListNode:
+    def __init__(self, x):
+        self.value = x
+        self.next = None
+
+def kth_last_element(head, k):
```
Comment (ariava) [inline aliskin/assignment1/kth-element.py:6]: Can you add a few examples to show that the function really works? :)

**Review by ariava — CHANGES_REQUESTED** [review]

**Comment by aliskin** [comment]
Now the examples are there :)

**Review by ariava — APPROVED** [review]
Super!

**Pull request merged**